engine/swarm/configs.md

```diff
@@ -612,3 +612,53 @@ configuration file.
 
 You have now updated your `nginx` service's configuration without the need to
 rebuild its image.
+
+### Example: Use templated config in Nginx service
+
+To create a configuration in which the content will be generated using a
+template engine, use the `--template-driver` parameter and specify the engine
+name as its argument. The template will be rendered when container is created.
```

Comment:
Something for a follow-up; we should add a list of placeholders that can be used in the reference docs. The `service create` page has a list: https://docs.docker.com/engine/reference/commandline/service_create/#create-services-using-templates, but templated configs can also reference secrets (see https://github.com/moby/moby/pull/33702), and I don't think that's currently documented

Reply:
Yes, it's not documented